src/box/box.cc

```diff
@@ -240,6 +248,18 @@ static char box_feedback_host[BOX_FEEDBACK_HOST_MAX];
 /** Whether sending crash info to feedback URL is enabled. */
 static bool box_feedback_crash_enabled;
 
+double
+tarantool_uptime(void)
+{
+	return ev_monotonic_now(loop()) - (ev_tstamp)tarantool_start_time;
+}
+
+sigint_cb_t __attribute__((weak))
+set_sigint_cb(sigint_cb_t new_sigint_cb)
+{
+	return new_sigint_cb;
+}
+
```

Comment:
I don't understand why these changes are needed and how they're connected with #8147.